test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -58,11 +62,15 @@ def test_smdebug_cpu(training, ec2_connection, region, cpu_only, py3_only):
         hide=True,
     )
 
-    ec2_connection.run(
+    test_output = ec2_connection.run(
         f"docker exec --user root smdebug-cpu /bin/bash -c '{test_script} {framework}'",
-        hide=True,
+        hide=True, warn=True
     )
 
+    LOGGER.info(test_output.stdout)
+
+    assert test_output.ok, "smdebug tests failed"
```

Comment:
Can we add more info to the assertion error? perhaps logging stdout and stderr would be useful

Reply:
Sure.